src/docfx/docfx.csproj

```diff
@@ -8,6 +8,7 @@
     <RootNamespace>Microsoft.Docs.Build</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PackAsTool>true</PackAsTool>
+    <NoWarn>NU1701</NoWarn>
```

Comment:
can this only apply to `HtmlAgilityPack`?

Reply:
sadly, no